content/en/docs/components/pipelines/v2/quickstart.md

```diff
@@ -1,16 +1,34 @@
 +++
```

Comment:
Global changes:

- Avoided forming possessives with product or technology names.
- First person narrative replaced with second person (for instructions) or (if needed) third person